client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -74,26 +81,49 @@ const MediaModalImageEditorCanvas = React.createClass( {
 	},
 
 	toBlob( callback ) {
-		const canvas = ReactDom.findDOMNode( this.refs.canvas );
-
-		MediaUtils.canvasToBlob( canvas, callback, this.props.mimeType, 1 );
+		const canvas = ReactDom.findDOMNode( this.refs.canvas ),
+			context = canvas.getContext( '2d' ),
```

Comment:
We prefer that each `const` and `let` declarations are on their own lines. It makes things a bit easier to refactor.
```javascript
const foo = 'foo';
const bar = 'bar';
const baz = 'baz';
```